main.go

```diff
@@ -292,7 +333,7 @@ func mainErr(args []string) error {
 			modpath, err := exec.Command("go", "list", "-m").Output()
 			if err == nil {
 				path := string(bytes.TrimSpace(modpath))
-				envGoPrivate = path+","+path+"_test"
+				envGoPrivate = path + "," + path + "_test"
```

Comment:
Revert

Reply:
I think this is because I forgot to run gofmt

Reply:
Yeah I didn't explicitly do that gofmt did